src/helpers/verification.js

```diff
@@ -21,6 +21,26 @@ exports.verifyNetlifyBuildVersion = ({ IS_LOCAL, NETLIFY_BUILD_VERSION, failBuil
   }
 }
 
+exports.checkForOldFunctions = async ({ functions }) => {
+  const oldFunctions = (await functions.list()).filter(({ name }) => name.startsWith('next_'))
+  if (oldFunctions.length !== 0) {
+    console.log(
+      yellowBright(outdent`
+      We have found the following functions in your site that seem to be left over from the old Next.js plugin. We have guessed this because the name starts with "next_".
```

Comment:
```suggestion
      We have found the following functions in your site that seem to be left over from the old Next.js plugin (v3). We have guessed this because the name starts with "next_".
```